commit 3c4429f5dd2c6f9665324f3ecadea69a0b6ad89b
Author: freedesktop_sdk_updater <freedesktop_sdk_updater@libreml.com>
Date:   Sat Feb 22 17:02:03 2025 +0000

    Update elements/include/linux.yml to v6.13.4-0
    
    Updates elements/include/linux.yml to v6.13.4-0 (24177f2)
    Previous version was v6.13.2-0 (0da4b4b)

diff --git a/elements/include/linux.yml b/elements/include/linux.yml
index 438d5ab28..36075e213 100644
--- a/elements/include/linux.yml
+++ b/elements/include/linux.yml
@@ -4,7 +4,7 @@ sources:
   track: v*
   exclude:
   - '*-rc*'
-  ref: v6.13.2-0-g0da4b4b84ffe5efe12f5fd96dd294aa9d20ba2de
+  ref: v6.13.4-0-g24177f243e34eef50b5ff4a656a622864fa177c2
 - kind: local
   path: files/linux/config-utils.sh
 - kind: local
